libbpf-rs/src/map.rs

```diff
@@ -171,6 +171,7 @@ pub struct Map {
     key_size: u32,
     value_size: u32,
     max_entries: u32,
+    id: u32,
```

Comment:
I am not opposed to this addition, just curious why you think it's needed? Given the `MapInfo` API we could always query it on demand, right? That seems preferable to duplicating members needlessly.

Similar comment about `max_entries`, actually.